apps-rendering/src/renderer.ts

```diff
@@ -200,10 +202,14 @@ const textElement =
 				}
 
 				const isFirstParagraph = node.previousSibling === null;
+				const isAfterDinkus =
+					node.previousSibling?.previousSibling?.textContent ===
```

Comment:
Would the dinkus (love that word :p ) always be two previous siblings away?

Reply:
Yes, there's always a line break in between. I followed this example: https://developer.mozilla.org/en-US/docs/Web/API/Node/previousSibling#second_example